var/spack/repos/builtin/packages/ratel/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ratel(MakefilePackage, CudaPackage, ROCmPackage):
+    """Extensible, performance-portable solid mechanics with libCEED and PETSc"""
+
+    homepage = "https://ratel.micromorph.org"
+    git = "https://gitlab.com/micromorph/ratel.git"
+
+    maintainers = ['jedbrown', 'jeremylt']
+
+    # development version
+    version('develop', branch='main')
+    depends_on('libceed@develop', when='@develop')
+    depends_on('petsc@main', when='@develop')
+
+    # version 0.1.1
+    version('0.1.1', tag='v0.1.1')
```

Comment:
`version` directives are grouped at the top in all other packages.

Reply:
I'm not sure what change you are requesting.. Do you just want me to delete the extra line and comment?

Reply:
No, just don't interleave `version` with `depends_on`. It's sort of a de facto style for packages.